web/src/utils/filter-options.ts

```diff
@@ -6,6 +6,7 @@ import { splitResource, SplitStage } from '../model/resource';
 import { autoCompleteCache } from './autocomplete-cache';
 import { DNS_RCODES } from './dns';
 import { getPort, getService } from './port';
+import { DROP_CAUSES, DROP_STATES } from './tcp-drop';
```

Comment:
pls rename the pkg to pkt-drop instead ?

Reply:
https://github.com/netobserv/network-observability-console-plugin/pull/362/commits/c29eb9f68ffddf4e92f26db9ab8ed802d2788be8